packages/2d/src/components/Shape.ts

```diff
@@ -69,6 +73,12 @@ export abstract class Shape extends Layout {
     context.lineCap = this.lineCap();
     context.setLineDash(this.lineDash());
     context.lineDashOffset = this.lineDashOffset();
+    if (!this.antialiased()) {
+      // from https://stackoverflow.com/a/68372384
+      context.imageSmoothingEnabled = false;
```

Comment:
I think we should merge `Shape.antialiased` and `Image.smoothing` together. They both control the `imageSmoothingEnabled` option and `Image` extends `Rect` which extends `Shape` so it will now have two duplicated properties.